rpc/transaction_test.go

```diff
@@ -265,9 +306,9 @@ func TestTransactionReceipt(t *testing.T) {
 			{
 				TxnHash: utils.TestHexToFelt(t, "0x49728601e0bb2f48ce506b0cbd9c0e2a9e50d95858aa41463f46386dca489fd"),
 				ExpectedResp: TransactionReceiptWithBlockInfo{
-					TransactionReceipt: receiptTxnIntegration,
-					BlockNumber:        319132,
-					BlockHash:          utils.TestHexToFelt(t, "0x50e864db6b81ce69fbeb70e6a7284ee2febbb9a2e707415de7adab83525e9cd"),
+					UnknownTransactionReceipt: UnknownTransactionReceipt{receiptTxnIntegration},
```

Comment:
This will need updated as well, but we can leave that for a seperate PR and focus on testnet updates if you'd like

Reply:
Actually, would you mind updating this as well. That way we can mark transaction_test.go as no longer depending on goerli test data